tests/test_cases/close.c

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <errno.h> 
+
+#define FILE_PATH "close.txt"
```

Comment:
is it better to generate a random name? as some other test could interfere with this?